docs/source/configuration/overview.mdx

```diff
@@ -537,6 +541,23 @@ preview_operation_limits:
 
 See [this article](./operation-limits/).
 
+#### Network-based limits
+
+##### `http_max_request_bytes`
+
+> **This configuration is currently [experimental](/resources/product-launch-stages#experimental-features).**
+
+Limits the amount of data read from the network for the body of HTTP requests,
+to protects against unbounded memory consumption.
+This limit is checked before JSON parsing.
+Both the GraphQL document and associated variables count toward it.
+
+The default value is `2000000` bytes, 2 MB.
+
+Before increasing this limit significantly consider doing performance testing
+in an environment similar to your production, especially if some clients are untrusted.
+Many concurrent large requests could can the Router to run out of memory.
```

Comment:
@chandrikas in case you want to have a look at the wording here. (The same is copied in the changelog.)